JapaneseFilter/sections/adservers.txt

```diff
@@ -65,6 +65,7 @@
 !
 ||speee-ad.akamaized.net^$third-party
 ||candy.smv.to^$third-party
+||ad999.biz^$third-party
```

Comment:
To mobile filter/adservers, only seems to be triggered when visiting the site on mobile device.

Ad left-overs on desktop page:

geeksokuhou.net##.main-image-in > .blogroll-wrapper + table[width]:not([id]):not([class])
geeksokuhou.net#?#.main-image-in > .blogroll-wrapper:has(> .blogroll-column > b > script:contains(blogroll_channel_id))

Ad on mobile:

geeksokuhou.net##a[href^="https://link-a.net/gate.php"]


Looks also like ad to me on http://geeksokuhou.net/archives/282558.html on mobile:

section#text-28